src/SWP/Bundle/ContentBundle/EventListener/ArticleMetadataSubscriber.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
```

Comment:
I wanted to separate metadata handling. I might change it. I can maybe add a getter in the package class which will return me such array of needed metadata and in ArticleFactory just do `$article->setMetadata($package->getMetadata())`